uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/Service/LeaveRequestCommentTest.php

```diff
@@ -175,20 +176,268 @@ public function testGetReturnsAssociatedCommentsForLeaveRequest() {
     $this->assertEquals($expected1, $result['values']);
   }
 
+  public function testGetReturnsOnlyCommentsOfLeaveRequestsOfTheCurrentLoggedInUserWhenTheLoggedInUserIsAStaffMember() {
+    $entityName = 'LeaveRequest';
+
+    $contact1 = ContactFabricator::fabricate();
+    $contact2 = ContactFabricator::fabricate();
```

Comment:
We don't really need Contact fabricators here. We can just use static ids. Will save some test time.